### DISCUSSION TITLE
Keep selected option at the top of the options list.

### DISCUSSION THREAD

**Discussion by arek-kilimar-allhuman** [selectize/selectize.js#1871] [Q&A]
I'm trying to keep chosen option at the top of the list.
I used object like this
    const countryCodes = [
        {
            "country": "Afghanistan",
            "pref": "+93",
            "code": "AF",
            "priority": 1
        }, 
        ........
adding "priority" set to one for each one and for each onChange I assign lower priority value to the chosen option. 

Then I sorted it like this  - sortField: ['priority','country']

It works kind of well, but the chosen option shows as the first one only when I choose next one and there is always one step delay.

I thought refreshOptions() would help here, but it doesn't.

Is there any way to make it working properly

**Comment by risadams** [comment]
A PR for this was just merged today,  it will be available once the next version is released.
you will need to set the option `setFirstOptionActive` to `true` and it should have the behavior you need.

**Reply by arek-kilimar-allhuman** [reply]
That's great, thanks. Do you know when approximately will be the next release?

I tried to make it with adding priority field, like here [jsfiddle](https://jsfiddle.net/ArekKDev/phkuz4Lb/3/) but it always appears at the top only when the next option is chosen. 

Maybe you have an idea how to make it working, before the next release.
